tests/cli/test_datasource_pandas.py

```diff
@@ -8,7 +8,7 @@
 
 from great_expectations import DataContext
 from great_expectations.cli import cli
-from tests.cli.utils import assert_no_logging_messages_or_tracebacks
+from tests.cli.utils import assert_no_logging_messages_or_tracebacks, escape_ansi
```

Comment:
Our testing for CLI output is incredibly granular and is a bit overkill in my opinion. I've added this `escape_ansi` helper to strip expected and actual outputs so we can assert the important textual content. Colors are helpful but I think validating hundreds of ANSI codes doesn't provide any real value.

Let me know if you disagree!